Makefile

```diff
@@ -54,7 +54,7 @@ RESTY_LMDB ?= `grep RESTY_LMDB $(KONG_SOURCE_LOCATION)/.requirements | awk -F"="
 LIBYAML_VERSION ?= `grep ^LIBYAML_VERSION $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
 RESTY_WEBSOCKET ?= `grep RESTY_WEBSOCKET $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
 OPENRESTY_PATCHES ?= 1
-DOCKER_KONG_VERSION = 'master'
```

Comment:
workaround to the microdnf issue that currently exists when using docker-kong@master

```
 Step 12/20 : RUN set -ex;     if [ "$ASSET" = "remote" ] ; then       VERSION=$(sed -n -e 's/^VERSION_ID="\(.*\)\.[0-9]"/\1/p' /etc/os-release)       && DOWNLOAD_URL="[https://download.konghq.com/gateway-${KONG_VERSION%%.*}.x-rhel-$VERSION/Packages/k/kong-$KONG_VERSION.rhel$VERSION.amd64.rpm](https://download.konghq.com/gateway-$%7BKONG_VERSION%%.*%7D.x-rhel-$VERSION/Packages/k/kong-$KONG_VERSION.rhel$VERSION.amd64.rpm)"       && curl -fL $DOWNLOAD_URL -o /tmp/kong.rpm       && echo "$KONG_SHA256  /tmp/kong.rpm" | sha256sum -c -;     fi     && microdnf install --assumeyes --nodocs       findutils       shadow-utils       unzip     && rpm -qpR /tmp/kong.rpm       | grep -v rpmlib       | xargs -n1 -t microdnf install --assumeyes --nodocs     && rpm -iv /tmp/kong.rpm     && microdnf -y clean all     && rm /tmp/kong.rpm     && chown kong:0 /usr/local/bin/kong     && chown -R kong:0 /usr/local/kong     && ln -s /usr/local/openresty/bin/resty /usr/local/bin/resty     && ln -s /usr/local/openresty/luajit/bin/luajit /usr/local/bin/luajit     && ln -s /usr/local/openresty/luajit/bin/luajit /usr/local/bin/lua     && ln -s /usr/local/openresty/nginx/sbin/nginx /usr/local/bin/nginx     && kong version

 ---> Running in 54173f469a03

+ '[' local = remote ']'

+ microdnf install --assumeyes --nodocs findutils shadow-utils unzip

/bin/sh: microdnf: command not found
```